### DISCUSSION TITLE
Only getting single channel with Behringer UMC202HD

### DISCUSSION THREAD

**Discussion by ctibke** [jamulussoftware/jamulus#1533] [Technical Support & Help] [answered]
Hi all,

newby to Jamulus which our choir would love to use for virtual sessions I acquired a condensor Microphone (Marantz MPM 1000 with XLR) and a Behringer UMC202HD which got good feedback on the community list https://jamulus.io/kb/2021/01/05/Jamulus-Sound-Devices.html

however I cannot find a way to produce output to both left and right channel output / mono towards my Windows 10 laptop. The Behringer interface has two XLR inputs which are treated as seperate channels, but the UMC Control Panel only seems to allow the 2 channel stereo output. Neither can I find an option via the ASIO4all driver to treat the UMC as line input

Anyone who'd have advice how to work around this issue?

many thanks
Carsten

**Comment by dcorson-ticino-com** [comment, marked as answer]
Hi Carsten,
The two inputs on your interface are thought for a stereo signal, left and right.
If you only have one mic you will only be using one channel.
To put that on both channels choose Mono-in/Stereo-out in Jamulus' settings. 
BTW: prefer the Behringer ASIO driver to ASIO4ALL, you will get better performance.

**Comment by ctibke** [comment]
Hi there,

I was sure I had tried that, but have to concede now it seems to work for the general output without issues. Sigh...

many thanks, also for the fast reaction ;-)

Carsten

**Reply by dakhubgit** [reply]
There is a left/right balance slider where you can in effect mute the irrelevant channel when sending in mono.  However, it is oriented and labeled and documented in a maximally confusing manner (and until recently, the documentation did not agree with the behavior) so that it is comparatively easy to shift it to the wrong direction for muting the channel you don't use.  In that case, you get only the inactive channel sampled and sent.  So when in doubt, leave it in the middle until you manage to get output and only then try out which of the extreme positions would be the one you want.
